tests/test_io.py

```diff
@@ -642,3 +642,328 @@ def test_default_audio_subtype(default_audio_subtype_mock):
     subtype_return = pyfar.io.default_audio_subtype(format=audio_format)
     assert subtype_return == 'bla'
     default_audio_subtype_mock.assert_called_with(audio_format)
+
+
```

Comment:
How about moving the new tests to test_io_comsol.py to make them easier to locate?